src/app/component/header/header.component.css

```diff
@@ -1,30 +1,46 @@
 
-.head {
+.linguist-title {
     font-family: "Pridi", serif;
     font-weight: 500;
     font-size: 50px;
   }
 
-.head2{
+.linguist-subtitle{
     font-family: "Pridi", serif;
     font-weight: 500;
-    font-size: 14px;
-    padding-left:14%;
+    font-size: 13px;
+    z-index: 1;
+    position:relative;
+    left:53% ;
+    
 }
 
-.search{
-    margin-left: auto;
+.linguist-search{
+    margin: 0;
+    padding: 0;
 }
 
 
-.div1{
-    padding: 40px 0px 0px 0px ;
+.linguist-header{
+    padding: 30px;
+    margin: 20px;
     display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-radius: 20px;
     flex-direction: row;
     align-items: center;
-    height: 50px;
+    background-color: rgb(236, 126, 57);
```

Comment:
Remove color -- keep it black and white.

Reply:
done